src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+  const rentCar = 40 * days;
+
+  if (days >= 7) {
+    discount = 50;
+
+    return rentCar - discount;
+  }
+
+  if (days >= 3 && days < 7) {
+    discount = 20;
+
+    return rentCar - discount;
+  }
```

Comment:
so you solution will looks like 
```suggestion
  if (days >= SOME_VAR && days < ANOTHER_VAR) {
    return rentCar - ONE_MORE_VAR;
  }
```